pwiz_tools/Skyline/SettingsUI/PeptideSettingsUI.cs

```diff
@@ -85,10 +85,31 @@ public class QuantificationTab : IFormView { }
         private static readonly IList<int?> _quantMsLevels = ImmutableList.ValueOf(new int?[] {null, 1, 2});
         private readonly LabelTypeComboDriver _driverSmallMolInternalStandardTypes;
 
-        public PeptideSettingsUI(SkylineWindow parent, LibraryManager libraryManager)
+        public PeptideSettingsUI(SkylineWindow parent, LibraryManager libraryManager, TABS? selectTab)
         {
             InitializeComponent();
 
+            if (selectTab == null)

```

Comment:
This is also already a pretty long function, and I would prefer to see the two switch statements closer together. Could we implement RestoreTabSel(TABS? selectTab) and SaveTabSel() which just get called by the constructor and the OnFormClosing(), and put the two functions right next to each other.

Reply:
Done.